modules/core/src/v2/coins/trx.ts

```diff
@@ -202,11 +220,49 @@ export class Trx extends BaseCoin {
   signMessage(key: KeyPair, message: string | Buffer): Buffer {
     const toSign = this.toHexString(message);
 
-    let sig = tronWeb.Trx.signString(toSign, key.prv, true);
+    let sig = bitgoAccountLib.Trx.Utils.signString(toSign, key.prv, true);
 
     // remove the preceding 0x
     sig = sig.replace(/^0x/, '');
 
     return Buffer.from(sig, 'hex');
   }
+
+  /**
+   * Explain a Tron transaction from txHex
+   * @param params
+   * @param callback
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TronTransactionExplanation>
+  ): Bluebird<TronTransactionExplanation> {
+    return co<TronTransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.halfSigned && params.halfSigned.txHex);
+      if (!txHex || !bitgoAccountLib.Trx.Utils.isValidHex(txHex)) {
+        throw new Error('missing valid txHex string');
+      }
+      assert(params.fee && params.txID);
+      const rawContract = bitgoAccountLib.Trx.Utils.decodeRawTransaction(txHex);
```

Comment:
This is not gonna work. The txHex is a stringfied JSON, like `'{\"some\":\"value\"}'`, you have to call `JSON.parse(txHex)` to get the JSON. Then run `bitgoAccountLib.Trx.Utils.decodeTransaction(tx.raw_data_hex)`

Reply:
Note to Gloria:
"{//df}" -> sendTransaction JSON -> use get methods in the library